io.edgehog.devicemanager.config.Telemetry.json

```diff
@@ -0,0 +1,24 @@
+{
+    "interface_name": "io.edgehog.devicemanager.config.Telemetry",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "datastream",
+    "ownership": "server",
+    "mappings": [
+        {
+            "endpoint": "/request/enable",
+            "type": "boolean",
+            "description": "Enable/Disable telemetry routine"
+        },
+        {
+            "endpoint": "/request/periodMS",
```

Comment:
We should use parametric properties, such as:
`/foobar/%{interface_name}/`, so we can have: `/foobar/io.edgehog.devicemanager.WiFiScanResults/periodMS`.

Reply:
let's not use milliseconds, it would be dangerous sending updates every 1 millisecond, just change it to seconds.